internal/pkg/cli/flag.go

```diff
@@ -218,6 +221,9 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	deleteSecretFlagDescription      = "Deletes AWS Secrets Manager secret associated with a pipeline source repository."
 	svcPortFlagDescription           = "The port on which your service listens."
 
+	noSubscriptionFlagDescription = "Optional. Turn off selection for adding subscriptions for worker services."
+	subscribeFlagDescription      = "Optional. Publishers for worker services to subscribe to. Must be of format '<svcName>:<publisherName>"
```

Comment:
🙏 
```suggestion
	subscribeFlagDescription      = `Optional. Events to subscribe to from services in your application. 
Must be of format '<svcName>:<topicName>'`
```
(There is a missing quote at the end)
(Line is longer than 80 chars can we split into multiple lines).

Reply:
Do you think it's worthwhile to namespace our command or values to mention "topics" in order to accomodate for "eventbridge" in the future?

Some options:
```
--topic-subscriptions  Optional. Topics from other services in your application to subscribe to.
                       Must be of the format '<svcName>:<topicName>'
```
or 
```
--subscribe   Optional. Events to subscribe to from services in your application. 
              Format for SNS topics: '<svcName>:topic:<topicName>'.
              Format for EventBus: 'eventbus:<details>'
```